morphic.js

```diff
@@ -9403,16 +9439,34 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        var w = this.parentThatIsA(WorldMorph);
```

Comment:
This should be `world` (I should have caught that earlier...)
Reason: Other functions that need a world reference use `world`